dumpling/export/dump_test.go

```diff
@@ -312,7 +312,7 @@ func TestSetSessionParams(t *testing.T) {
 		WillReturnError(tikvErr)
 
 	require.NoError(t, failpoint.Enable("github.com/pingcap/tidb/dumpling/export/SkipResetDB", "return(true)"))
```

Comment:
You can try `testfailpoint.Enable`

Reply:
seems not work, will raise the same error

Reply:
Can you provide your diff about `testfailpoint.Enable`? We use it elsewhere and I don't understand why it will not work